src/policy/vmspace.rs

```diff
@@ -224,6 +224,14 @@ impl<VM: VMBinding> VMSpace<VM> {
             start: start.align_down(BYTES_IN_PAGE),
             end: end.align_up(BYTES_IN_PAGE),
         });
+
+        if self.common.needs_log_bit {
```

Comment:
Another concern is that this approach assumes the VM binding cannot set the unlog bits properly for reasons such as not being able to enumerate objects in the boot image.  This shouldn't be the default.  Instead, we can make it an option to bulk set side unlog bits.  It should be guarded by a Cargo feature so that the user knows it is a workaround.  (Alternatively, it can also be guarded by a constant in `ObjectModel`.  The bulk-setting operation can also be an API for the binding to invoke explicitly.)

Reply:
I've made it a feature